CMSIS/Core/Test/lit.cfg.py

```diff
@@ -612,6 +612,374 @@
+        'features': ['armv7r', 'thumb-2', 'sat', 'dsp', 'clz'],
+        'header': 'core_cr5.h',
+        'defines': {
+            '__CR5_REV': '0x0000U',
+            '__FPU_PRESENT': '1U',
+            '__GIC_PRESENT': '1U',
+            '__TIM_PRESENT': '1U',
+            '__L2C_PRESENT': '1U',
+            '__MPU_PRESENT': '1U',
+            'GIC_DISTRIBUTOR_BASE': '0x2C001000UL',
+            'GIC_INTERFACE_BASE': '0x2C000100UL',
+            'TIMER_BASE': '0x2C000600UL',
+            'L2C_310_BASE': '0x2C0F0000UL',
+            'IRQn_Type': 'int'
+        }
+    },
+    'CR7': {
+        'arch': 'armv7r',
+        'triple': 'armv7-r',
+        'abi': 'eabi',
+        'mcpu': 'cortex-r7',
+        'mfpu': 'none',
+        'mpu': True,
+        'features': ['armv7r', 'thumb-2', 'sat', 'clz'],
+        'header': 'core_cr7.h',
+        'defines': {
+            '__CR7_REV': '0x0000U',
+            '__FPU_PRESENT': '0U',
+            '__GIC_PRESENT': '1U',
+            '__TIM_PRESENT': '1U',
+            '__L2C_PRESENT': '1U',
+            '__MPU_PRESENT': '1U',
+            'GIC_DISTRIBUTOR_BASE': '0x2C001000UL',
+            'GIC_INTERFACE_BASE': '0x2C000100UL',
+            'TIMER_BASE': '0x2C000600UL',
+            'L2C_310_BASE': '0x2C0F0000UL',
+            'IRQn_Type': 'int'
+        }
+    },
+    'CR7F': {
+        'arch': 'armv7r',
+        'triple': 'armv7-r',
+        'abi': 'eabi',
+        'mcpu': 'cortex-r7',
+        'mfpu': 'VFPv3-D16',
+        'mpu': True,
+        'features': ['armv7r', 'thumb-2', 'sat', 'dsp', 'clz'],
+        'header': 'core_cr7.h',
+        'defines': {
+            '__CR7_REV': '0x0000U',
+            '__FPU_PRESENT': '1U',
+            '__GIC_PRESENT': '1U',
+            '__TIM_PRESENT': '1U',
+            '__L2C_PRESENT': '1U',
+            '__MPU_PRESENT': '1U',
+            'GIC_DISTRIBUTOR_BASE': '0x2C001000UL',
+            'GIC_INTERFACE_BASE': '0x2C000100UL',
+            'TIMER_BASE': '0x2C000600UL',
+            'L2C_310_BASE': '0x2C0F0000UL',
+            'IRQn_Type': 'int'
+        }
+    },
+    'CR8': {
+        'arch': 'armv7r',
+        'triple': 'armv7-r',
+        'abi': 'eabi',
+        'mcpu': 'cortex-r8',
+        'mfpu': 'none',
+        'mpu': True,
+        'features': ['armv7r', 'thumb-2', 'sat', 'clz'],
+        'header': 'core_cr8.h',
+        'defines': {
+            '__CR8_REV': '0x0000U',
+            '__FPU_PRESENT': '0U',
+            '__GIC_PRESENT': '1U',
+            '__TIM_PRESENT': '1U',
+            '__L2C_PRESENT': '1U',
+            '__MPU_PRESENT': '1U',
+            'GIC_DISTRIBUTOR_BASE': '0x2C001000UL',
+            'GIC_INTERFACE_BASE': '0x2C000100UL',
+            'TIMER_BASE': '0x2C000600UL',
+            'L2C_310_BASE': '0x2C0F0000UL',
+            'IRQn_Type': 'int'
+        }
+    },
+    'CR8F': {
+        'arch': 'armv7r',
+        'triple': 'armv7-r',
+        'abi': 'eabi',
+        'mcpu': 'cortex-r8',
+        'mfpu': 'VFPv3-D16',
```

Comment:
Value for `mpfu` should be lower case:

```suggestion
        'mfpu': 'vfpv3-d16,
```